lib/generators/draft/resource/templates/specs/factories.rb

```diff
@@ -19,15 +18,15 @@
 <% when :decimal -%>
     <%= attribute.column_name %> 42.42
 <% when :integer, :primary_key -%>
```

Comment:
We may not need the `:primary_key` part anymore, since the db will do it automatically?

Reply:
Do you remember what it was used for in the first place? I'm not sure `increasing_random.rb` would have been dependent on this option.

Reply:
I actually don't remember why I put this in. And there's no helpful commit message 😔

In any case I don't expect our students to ever use `:primary_key` when generating resources so this must have been for future-proofing. I'd lean towards removing it and then adding it back iff we run into a need. Thoughts?

Reply:
Sounds great. I just wanted to make sure we weren't going to be inadvertently causing problems down the line with removal.